install_script.sh.template

```diff
@@ -651,22 +658,24 @@ elif [ "$OS" = "SUSE" ]; then
       agent_version_custom="$(zypper search -s datadog-agent | grep -E "$pkg_pattern" -om1)" || true
       verify_agent_version "-"
   fi
-  echo -e "  \033[33mInstalling package: $agent_flavor\n\033[0m"
 
   declare -a packages
   packages=("$agent_flavor")
   if [ -n "$fips_mode" ]; then
     packages+=("datadog-fips-proxy")
   fi
 
+
+  echo -e "  \033[33mInstalling package(s): ${packages[*]}\n\033[0m"
+
   if [ -z "$sudo_cmd" ]; then
     ZYPP_RPM_DEBUG="${ZYPP_RPM_DEBUG:-0}" zypper --non-interactive install "${packages[@]}" ||:
   else
     $sudo_cmd ZYPP_RPM_DEBUG="${ZYPP_RPM_DEBUG:-0}" zypper --non-interactive install "${packages[@]}" ||:
   fi
 
   # If zypper fails to refresh a random repo, it installs the package, but then fails
-  # anyway. Therefore we let it do its thing and then see if curl was installed or not.
+  # anyway. Therefore we l-    declare -a packages
```

Comment:
This seems to be a copy-paste error?

Reply:
Indeed, must've fumbled my fingers with vim shortcuts.